src/views/workflow-history/workflow-history-events-card/workflow-history-events-card.styles.ts

```diff
@@ -0,0 +1,65 @@
+import { type Theme } from 'baseui';
+import { type AccordionOverrides } from 'baseui/accordion';
+import { type StyleObject } from 'styletron-react';
+
+import type {
+  StyletronCSSObject,
+  StyletronCSSObjectOf,
+} from '@/hooks/use-styletron-classes';
+
+export const overrides = {
+  accordion: {
+    Root: {
+      style: ({ $theme }: { $theme: Theme }): StyleObject => ({
+        ...$theme.borders.border100,
+        borderRadius: $theme.borders.radius300,
+        overflow: 'hidden',
+      }),
+    },
+    Header: {
+      style: ({ $theme }: { $theme: Theme }): StyleObject => ({
+        gap: $theme.sizing.scale500,
+        paddingTop: $theme.sizing.scale400,
+        paddingRight: $theme.sizing.scale600,
+        paddingBottom: $theme.sizing.scale400,
+        paddingLeft: $theme.sizing.scale500,
+      }),
+    },
+    ToggleIcon: {
+      style: ({
+        $theme,
+        $disabled,
+      }: {
+        $theme: Theme;
+        $disabled: boolean;
+      }): StyleObject => ({
+        height: $theme.sizing.scale600,
+        width: $theme.sizing.scale600,
+        opacity: $disabled ? 0 : undefined,
```

Comment:
It's a bit strange explicitly defining opacity as undefined. I'd rather do it like this:
```suggestion
        ...($disabled && {opacity: 0}),
```